Sources/Controllers/Map/Layers/OAMapLayer.h

```diff
@@ -21,8 +21,11 @@
 @property (nonatomic, readonly) OAMapRendererView *mapView;
 @property (nonatomic, readonly) BOOL nightMode;
 @property (nonatomic, readonly) CGFloat displayDensityFactor;
+@property (nonatomic, readonly) int baseOrder;
```

Comment:
define pointsOrder similar to baseOrder
`@property (nonatomic, readonly) int pointsOrder;`